### DISCUSSION TITLE
Potentially compile to Roblox

### DISCUSSION THREAD

**Discussion by Milo123459** [bevyengine/bevy#4732] [Ideas]
Hey!

I know this is probably going to be an unpopular feature request. I wanted to work on a game and thought of publishing to Roblox and wanted to use Bevy as I am already familiar with it. It would be cool if it could be fully translated into a Roblox game.

Please leave your thoughts in this thread!

**Comment by bjorn3** [comment]
I did call that infeasible. As I understand it roblox is a game engine itself and forces it's own way of doing things on users. Furthermore roblox is programmable using lua. There is no way to compile rust to lua.
